src/main/scala/com/softwaremill/realworld/articles/ArticleDto.scala

```diff
@@ -16,17 +37,37 @@ case class Article(
     favoritesCount: Int,
     author: ArticleAuthor
 )
-object Article:
-  given articleEncoder: zio.json.JsonEncoder[Article] = DeriveJsonEncoder.gen[Article]
-  given articleDecoder: zio.json.JsonDecoder[Article] = DeriveJsonDecoder.gen[Article]
+object ArticleData:
+  given articleDataEncoder: zio.json.JsonEncoder[ArticleData] = DeriveJsonEncoder.gen[ArticleData]
+  given articleDataDecoder: zio.json.JsonDecoder[ArticleData] = DeriveJsonDecoder.gen[ArticleData]
+
+case class ArticleCreateData(
+    title: String,
+    description: String,
+    body: String,
+    var tagList: List[String]// TODO tagList is currently mandatory...
```

Comment:
why is it a `var`?

Reply:
I was trying to do something (but scala has won) :)
forgot to clean that, thanks